src/llmcompressor/modifiers/quantization/gptq/utils/gptq_wrapper.py

```diff
@@ -108,17 +123,20 @@ def compress(
 
         tick = time.time()
 
-        if hasattr(self.layer, "quantization_scheme"):
-            quant_scheme = self.layer.quantization_scheme
-            if quant_scheme.weights is not None:
-                # fetch latest correct scale and ZP relevant for any changes
-                # such as activation reordering
-                from compressed_tensors.quantization import (
-                    update_layer_weight_quant_params,
-                )
-
-                update_layer_weight_quant_params(self.layer)
-
+        # compute quantization parameters
+        if weight_fake_quant is not None:
+            scale = weight_fake_quant.scale
+            zero_point = weight_fake_quant.zero_point
+            dtype = weight_fake_quant.dtype
+            tensor_scheme = weight_fake_quant.qscheme in [
+                torch.per_tensor_affine,
+                torch.per_tensor_symmetric,
+            ]
+        else:  # weight_quant_args is not None
+            observer = MemorylessObserver(weight_quant_args)
+            scale, zero_point = observer(W)
```

Comment:
Could you add a comment noting we need this scale/zp update for activation reordering?

Reply:
In the future we'll only do this in the activation ordering case

Reply:
Added, in the future we'll only update for activation ordering